frontend/pages/dashboard/[id].js

```diff
@@ -370,46 +373,61 @@ export default function Dashboard() {
     );
 
     // Checking if any of the secret keys start with a number - if so, don't do anything
-    const nameErrors = !Object.keys(obj).map(key => !isNaN(key.charAt(0))).every(v => v === false);
-    const duplicatesExist = data?.map(item => item[2]).filter((item, index) => index !== data?.map(item => item[2]).indexOf(item)).length > 0;
+    const nameErrors = !Object.keys(obj)
+      .map((key) => !isNaN(key.charAt(0)))
+      .every((v) => v === false);
+    const duplicatesExist =
+      data
+        ?.map((item) => item[2])
+        .filter(
+          (item, index) => index !== data?.map((item) => item[2]).indexOf(item)
+        ).length > 0;
 
     if (nameErrors) {
-      console.log("Solve all name errors first!");
-    } else if (duplicatesExist) {
-      console.log("Remove the duplicated entries first!");
-    } else {
-      // Once "Save changes is clicked", disable that button
-      setButtonReady(false);
-      pushKeys({obj, workspaceId: router.query.id, env});
-
-      /**
-       * Check which integrations are active for this project and environment
-       * If there are any, update environment variables for those integrations
-       */
-      let integrations = await getWorkspaceIntegrations({
-        workspaceId: router.query.id,
+      return createNotification({
+        text: "Solve all name errors first!",
+        type: "error",
```

Comment:
I opted for early returns because they're easier to understand than if/else statements. Please let me know if you want this reverted.